filebeat/module/traefik/access/config/traefik-access.yml

```diff
@@ -7,9 +7,9 @@ exclude_files: [".gz$"]
 
 processors:
 - dissect:
-    tokenizer: '%{traefik.access.remote_ip} - %{traefik.access.user_name} [%{traefik.access.time}]
-    "%{traefik.access.method} %{traefik.access.url} HTTP/%{traefik.access.http_version}"
-    %{traefik.access.response_code} %{traefik.access.message}'
+    tokenizer: '%{traefik.access.remote_ip} - %{user.name} [%{traefik.access.time}]
+    "%{http.request.method} %{traefik.access.url} HTTP/%{http.version}"
+    %{http.response.status_code} %{traefik.access.message}'
```

Comment:
message?

Reply:
@ph Every time I look at dissect I'm so happy that I don't have to look at grok patterns ❤️

Reply:
Message here is used as "rest", which gets parsed by GROK ;-) in ingest node